content/javascript/concepts/accessors/accessors.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Accessors'
+Description: "Offers access to an object's properties through either dot notation or bracket notation."
```

Comment:
```suggestion
Description: 'Offers access to the properties of an object through either dot notation or bracket notation.'
```

Reply:
metadata must only use single quotes and there is no punctuation permitted within the description